src/lib/AppRegistry.tsx

```diff
@@ -67,6 +63,10 @@ const Gene: React.SFC<GeneProps> = ({ geneID, refineSettings: { medium, price_ra
   return <GeneRenderer {...initialProps} render={renderWithLoadProgress(Containers.Gene, initialProps)} />
 }
 
+const Home: React.SFC<{ selectedArtist: string }> = ({ selectedArtist }) => {
+  return <HomeScene selectedArtist={selectedArtist} />
+}
```

Comment:
I’m not sure I understand why this is needed, it looks like both components basically expose one and the same prop. Can you elaborate?

Reply:
The prop isn't passed down to `Home` without being this explicit.

Reply:
Actually I think I had something else missing before that was interfering here; this can be removed 😁

Reply:
Ah ok, I was still scratching my head about what could have been different :D